README.md

```diff
@@ -54,6 +54,18 @@ To use these artifacts, you must also depend on `io.opentelemetry:opentelemetry-
 See [opentelemetry-java releases](https://github.com/open-telemetry/opentelemetry-java#releases) for
 more information.
 
+> Android Requirements
+> 
+> If you are using this on Android and your project's minSdk is lower than 26, you must enable
```

Comment:
Can we just link to the core repo section which talks about desugaring? https://github.com/open-telemetry/opentelemetry-java/blob/main/VERSIONING.md#language-version-compatibility

The "minSdk is lower than 26" requirement is the type of thing what will become out of date and I'm not really sure where 26 comes from.

Reply:
26 is Android API version code. It's pretty impenetrable unless you're deep in that ecosystem.

I'll make a quick reference to desugaring here and point to the `opentelementry-android` repo for more details so we can isolate the details in one place at least.

I'll put up a revision in the next couple days.